mozilla/fx-private-relay
# MPP-3588: 2023 Holiday promo banner

This PR fixes MPP-3588.



# New feature description

A banner for our 2023 holiday promo was added for non signed in users on /premium and the relay home page.

Feature flag: holiday_promo_2023

# Demo

[Demo](https://github.com/mozilla/fx-private-relay/assets/59676643/4c63178f-6ece-47a3-933e-e00fc82c4ba5)

# How to test
1. Give yourself the waffle flag `holiday_promo_2023`
2. Go to the relay home page as a non-signed in user and verify the following:
- A user that is not signed in can see the banner on the relay home page (“/”) and on the premium page (“/premium”).
- A user that is signed in does not see the banner.
- Clicking the get 1 year of premium button takes the user to the Email 1 year plan and with the coupon code.
- Banner should be responsive to all screens.
- Banner is sticky, stays with the navbar.






# Checklist (Definition of Done)
